internal/endtoend/testdata/func_args/go/query.sql.go

```diff
@@ -45,12 +45,12 @@ SELECT plus(b => $2, a => $1)
 `
 
 type PlusParams struct {
-	B int32
 	A int32
+	B int32
 }
 
 func (q *Queries) Plus(ctx context.Context, arg PlusParams) (int32, error) {
-	row := q.db.QueryRowContext(ctx, plus, arg.B, arg.A)
+	row := q.db.QueryRowContext(ctx, plus, arg.A, arg.B)
```

Comment:
!!!

The compiler generated incorrect code here. The order of the arguments wasn't correct.